app/scripts/modules/netflix/serverGroup/awsServerGroupDetails.html

```diff
@@ -291,6 +291,16 @@ <h3 select-on-dbl-click>
             Cluster ({{ctrl.serverGroup.cluster}})
           </a>
         </li>
+        <li>
+          <a target="_blank" ng-href="http://edda-main.{{ctrl.serverGroup.region}}.{{ctrl.serverGroup.account}}.netflix.net:7001/REST/v2/aws/launchConfigurations/{{ctrl.serverGroup.launchConfig.launchConfigurationName}}">
```

Comment:
At some point we might/will get a complaint that the edda links don't work for anything but the prod/test accounts (edda-main vs. edda-mce).